executor/executor_test.go

```diff
@@ -6849,3 +6849,61 @@ func (s *testSuite) TestIssue19667(c *C) {
 	tk.MustExec("INSERT INTO t VALUES('1988-04-17 01:59:59')")
 	tk.MustQuery(`SELECT DATE_ADD(a, INTERVAL 1 SECOND) FROM t`).Check(testkit.Rows("1988-04-17 02:00:00"))
 }
+
```

Comment:
We could add some tests for patition table to verify the physical ids are used, and some tests for `union` statements using multiple data sources like `PointGetExec`.